paddle/fluid/platform/mkldnn_helper.h

```diff
@@ -99,5 +99,124 @@ inline mkldnn::memory::format GetMKLDNNFormat(const mkldnn::memory memory) {
       memory.get_primitive_desc().desc().data.format);
 }
 
+class MKLDNNHandler {
```

Comment:
Could you give more introductions about `MKLDNNHandler`？
Is this specifically added for softmax grad since [this](https://github.com/PaddlePaddle/Paddle/pull/11519/files#diff-92975572e9593f057bc7a77d0906ed7cR163).

When adding functions at `mkldnn_helper.h`, it should be something common I suppose.

Reply:
@tensor-tang You are right that softmax functionality should go to into MKLDNNHandler. MKLDNNHandler will agregate common functionality  as for softmax/conv/pool primitive specific functionality there will be designated derived classes eg. SoftmaxMKLDNNHandler , ConvMKLDNNHandler etc. I updated the code adding SoftmaxMKLDNNHandler. Please take a look